cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.1 on 2024-12-16 19:31
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cinema', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='movie',
+            name='image',
+            field=models.ImageField(null=True, upload_to='movie_image_path'),
```

Comment:
The `upload_to` parameter should reference the `movie_image_path` function directly, without quotes. Change `'movie_image_path'` to `movie_image_path` to ensure the upload path is set correctly.